README.md

```diff
@@ -149,7 +149,7 @@ For further details on the settings see: https://manual.limesurvey.org/Optional_
 
 # Running LimeSurvey with docker-compose
 
-The easiest way to get a fully featured and functional setup is using a docker-compose file. Several examples are provided in the [repository](https://github.com/martialblog/docker-limesurvey).
+The easiest way to get a fully featured and functional setup is using a docker-compose file. Several examples are provided in the [repository](https://github.com/martialblog/docker-limesurvey). To sign into the example admin panel the credentials are username: *admin* password: *foobar*.
```

Comment:
I like the idea, however, I think it would be better to have this as an extra paragraph and to explicitly mention that this is for testing only. The docker-compose files are only examples.

Reply:
Agreed, I was thinking about that as well, to explicitly state, or put a warning, that any use beyond running the example as-is, the person setting this up needs to define secure admin/password using environment variables as described in the Environment Variables section.

I'll write the paragraph and update for you to review.